### ISSUE TITLE
Remove All Toasts And Replace Them With Dialogs

### ISSUE THREAD

**Issue by ZacharyDavidSaunders** [Z-and-Z-Software/IT-Certification-Prep#42]
In order to keep a consistent user experience and improve look & feel, we should replace all toasts with dialogs.  This is another task that should be completed before the 1.0 release.

**Toast Example:**

![Toast](https://cloud.githubusercontent.com/assets/14244868/19525858/36814968-95d7-11e6-9141-43d70edcba55.png)

**Dialog Example:**

![Dialog](https://cloud.githubusercontent.com/assets/14244868/19526330/f4162a88-95d8-11e6-9446-97e6c862f6ae.png)

_**Development Instructions:**_

See https://developer.android.com/guide/topics/ui/dialogs.html and the dialogs in DrawerManager.java for an example of drawer implementation.

**Comment by Zackazt** [comment]
Pushed to Further-Enhance-Drawer-Menu-System, rather than creating a new branch for this issue.  Please verify that this is satisfactory.
